media-sound/teamspeak-server-bin/teamspeak-server-bin-3.1.1.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils multilib systemd user
+
+DESCRIPTION="TeamSpeak is software for quality voice communication via the Internet"
+HOMEPAGE="https://www.teamspeak.com/"
+SRC_URI="amd64? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_amd64-${PV}.tar.bz2 )
+	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_x86-${PV}.tar.bz2 )"
+
+LICENSE="teamspeak3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc mysql tsdns"
+
+RDEPEND="mysql? ( dev-db/mariadb-connector-c )"
```

Comment:
How is this used? I don't see mariadb driver in the archive. I honestly doubt it's going to try to load the system QtSql driver.

Reply:
Have a look at libts3db_mariadb.so. "ldd" shows a dependency for dev-db/mariadb-connector-c aka libmariadb.so.2.

But: I see, that I am using dev-db/mariadb-connector-c-2.3.5 (libmariadb.so.2) from my local overlay. Gentoo only has dev-db/mariadb-connector-c-3.0.3 (libmariadb.so.3). That means, this version is too new. Teamspeak depends on libmariadb.so.2.

Basically this means, mariadb support never has worked in gentoo "upstream" with teamspeak3-server. Seems, that the package was, before I took this for proxy maintain, already wrong.

So I see this options:
1) drop mariadb support
2) Include dev-db/mariadb-connector-c-2.3.5 in portage. It's still maintained. I've created a PR: https://github.com/gentoo/gentoo/pull/7499 Feel free to close, if this is not an option.
3) Teamspeak includes in the folder "redist" an libmariadb.so.2, but I am not happy to install a compiled lib..

What do you think?

Reply:
@gentoo/mysql maintains mysql. I suppose they wouldn't mind adding a compat `SLOT=2` or alike that just installs `.so` files for binary packages.

Reply:
Okay, let's see what @gentoo/mysql says..